### PR TITLE
10339 cleanup

### PR DESCRIPTION
**Pull request by nechama-krigsman**

### CODE REVIEW

**Review by cruzjone-flexion — APPROVED** [review]

File: cypress/helpers/fileAPetition/practitioner-creates-electronic-case.ts [hunk 1]
```diff
@@ -1,49 +1,5 @@
...
-  cy.get('[data-testid="complete-step-4"]').click();
-  cy.get('[data-testid="file-petition"]').click();
-  return cy
-    .get('[data-testid="docket-number-with-suffix"]')
-    .invoke('text')
-    .then(docketNumberWithSuffix => {
-      cy.get('[data-testid="button-back-to-dashboard"]').click();
-      return cy.wrap<string>(docketNumberWithSuffix);
-    });
+  return petitionerCreatesElectronicCase();
```
Comment (TomElliottFlexion) [inline cypress/helpers/fileAPetition/practitioner-creates-electronic-case.ts]: ExternalUserCreatesElectronicCase, perhaps?

File: shared/src/business/entities/EntityConstants.ts [hunk 2]
```diff
@@ -91,9 +91,6 @@ export const ALLOWLIST_FEATURE_FLAGS = {
   ENTITY_LOCKING_FEATURE_FLAG: {
     key: 'entity-locking-feature-flag',
   },
-  UPDATED_PETITION_FLOW: {
```
Comment (TomElliottFlexion) [inline shared/src/business/entities/EntityConstants.ts:94]: 🔥🔥🔥

File: web-client/integration-tests/journey/docketClerkUpdatesCaseCaption.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const docketClerkUpdatesCaseCaption = cerebralTest => {
     });
 
     expect(cerebralTest.getState('caseDetail.caseCaption')).toEqual(
-      'Daenerys Stormborn, Deceased, Daenerys Stormborn 2, Surviving Spouse, Petitioner',
```
Comment (TomElliottFlexion) [inline web-client/integration-tests/journey/docketClerkUpdatesCaseCaption.ts:10]: ...I kinda prefer Daenerys Stormborn over John Doe

Comment (TomElliottFlexion) [reply web-client/integration-tests/journey/docketClerkUpdatesCaseCaption.ts:10]: This comment is not meant to be taken seriously. Seriously.

**Review by TomElliottFlexion — APPROVED** [review]

File: cypress/helpers/fileAPetition/practitioner-creates-electronic-case.ts (hunk 1 above)
Comment (nechama-krigsman) [reply cypress/helpers/fileAPetition/practitioner-creates-electronic-case.ts]: Yes, will update. Thanks!

**Review by jimlerza — APPROVED** [review]

**Pull request merged**